smalltalksrc/VMMaker/SlangAbstractTestCase.extension.st

```diff
@@ -2,8 +2,9 @@ Extension { #name : #SlangAbstractTestCase }
 
 { #category : #'*VMMaker' }
 SlangAbstractTestCase >> setUp [
+
 	super setUp.
-	ccg := MLVMCCodeGenerator new.
+	ccg := MLVMCCodeGeneratorConflictMock new.
```

Comment:
Again a mock, is it really needed?

Reply:
Mocking was an easy way to not modify the flow of the existing tests. I am working on improving them.

the problem was that we add the class to test to Slang (and by doing so all of the methods) before each execution which now cause the inference of types to stop, leaving the remaining methods untreated. It made green tests go wrongfully red.

Reply:
I understand, but at the end we are testing a class that is not the real one.
Typically, when this happens, later we hit bugs and we say "But the tests are green!", and of course, because they test something different.

As I told you before, probably it's best to
 - group methods in different classes
 - add methods manually one by one on a by-need basis when adding a class is too much for a test